lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+'''
+Unit Tests for the manage.py command rename_user
+'''
+from django.core.management.base import CommandError
+import unittest
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from .. import rename_user
+import sys
+from StringIO import StringIO
+
+
+class TestUsernameRenameCommand(unittest.TestCase):
```

Comment:
I renamed this file to test_rename_user_command.py and class to TestRenameUserCommand. This is the command line tool which is different from the rename_user management utility. Now the filename for the test and the class name mimic one another